### PR TITLE
Introduce CSS file to skeleton project

### PR DESCRIPTION
**Pull request by rashadg1030**

### CODE REVIEW

File: skeleton/frontend/src/Frontend.hs [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
 import Common.Route
-import Obelisk.Generated.Static
+
 
 frontend :: Frontend (R FrontendRoute)
 frontend = Frontend
-  { _frontend_head = el "title" $ text "Obelisk Minimal Example"
+  { _frontend_head = do
+      el "title" $ text "Obelisk Minimal Example"
+      elAttr "link" ("href" =: (static @ "main.css") <> "type" =: "text/css" <> "rel" =: "stylesheet") blank
```
Comment (3noch) [inline skeleton/frontend/src/Frontend.hs]: `"href" =: (static @ "main.css")` can be `"href" =: static @"main.css"`

File: skeleton/frontend/src/Frontend.hs [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
 import Common.Route
-import Obelisk.Generated.Static
+
 
 frontend :: Frontend (R FrontendRoute)
 frontend = Frontend
-  { _frontend_head = el "title" $ text "Obelisk Minimal Example"
+  { _frontend_head = do
+      el "title" $ text "Obelisk Minimal Example"
+      elAttr "link" ("href" =: static @ "main.css" <> "type" =: "text/css" <> "rel" =: "stylesheet") blank
```
Comment (3noch) [inline skeleton/frontend/src/Frontend.hs]: The space between `@` and `"main.css"` is inconsistent with pattern below. I think we prefer no space so it's a bit more obvious that `@` is not an operator.

Comment (rashadg1030) [reply skeleton/frontend/src/Frontend.hs]: Right, my bad!

**Comment by 3noch** [comment]
Thanks! Sorry it took so long to get this in!

**Pull request merged**